app/models/chair.rb

```diff
@@ -0,0 +1,6 @@
+class Chair < ApplicationRecord
+  validates :name, presence: true
```

Comment:
In the model, we have a 1..*-relation to Person. Should we validate if there exists at least one person?

Reply:
Maybe something like `validates :person, presence: true` works?